docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+# Documentation editing guidelines
+
+This page covers guidelines for editing the [Handsontable documentation](https://handsontable.com/docs).
+
+## Maintenance rules
+
+When adding new documentation files, check the documentation [directory structure](./README.md#handsontable-docs-directory-structure), and follow the guidelines below.
+
+### Filenames
+
+* Use only lower-case characters.
+* To separate words, use `-`.
+* Use the `.md` file extension. The `.vue` extension is disabled.
+
+### Frontmatter
+
+Each Markdown file starts with the following frontmatter tags:
+
+| Tag            | Meaning                                                    | Required |
+|----------------|------------------------------------------------------------|----------|
+| `title`        | The page's header.                                         | Yes      |
+| `permalink`    | The page's **unique** URL.                                 | Yes      |
+| `canonicalUrl` | A canonical URL of the page's latest version.              | Yes      |
```

Comment:
Even if you don't add these tags - vuepress will generate pages with their parent's title and permalink as the file name (with `.html` extension). It's not a problem to me, but it's good to know that it works that way.

Reply:
From that purpose we use our linking syntax `@/path/to/file/in/current/version.md`
For instance: `@/api/pluginHooks.md` in scope of the `next` version makes: `/docs/next/hooks/`

Reply:
@swistach I added info about automatically-generated `title` and `permalink` tags: https://github.com/handsontable/handsontable/pull/8349